sdk/identity/azure-identity/README.md

```diff
@@ -201,18 +201,18 @@ ManagedIdentityCredential managedIdentityCredential = new ManagedIdentityCredent
         .build();
 
 ClientSecretcredential secondServicePrincipal = new ClientSecretCredentialBuilder()
-	    .clientId("<YOUR_CLIENT_ID>")
-	    .clientSecret("<YOUR_CLIENT_SECRET>")
-	    .tenantId("<YOUR_TENANT_ID>")
-	    .build();
+        .clientId("<YOUR_CLIENT_ID>")
+        .clientSecret("<YOUR_CLIENT_SECRET>")
```

Comment:
Is there some rule for indentation we should follow for code in READMEs?
Or just estimate it?

Reply:
This used to be a tab. It was converted to spaces.

Reply:
There was no indentation change. 1 tab = 4 spaces in our .editorconfig settings.

Reply:
oh GIthub shows it like this, like some extra space added here.
![image](https://user-images.githubusercontent.com/5430778/62653682-99796180-b94d-11e9-80e2-282c7f4ce819.png)

Reply:
Ahh, so before, they had one space, a tab character than then some spaces.

Reply:
so, replacing tab with 4 spaces added extra space then ?

Reply:
No. When you have a tab character, its rendered size depends on how large your tab size is. A tab size in our configuration is 4 spaces. So if you had a space then tab character, the tab character would only take up 3 spaces to fill in that first tab stop. If you had 0 spaces and pressed tab, it would take up the 4 spaces.